l2os/drivers/l2output/driver.go

```diff
@@ -205,8 +205,9 @@ func (d *Driver) CraftTx(
 	opts.Nonce = nonce
 	opts.NoSend = true
 
+	// TODO: Submit an actual l1 blockhash and blocknumber here for fork protection
 	return d.l2ooContract.AppendL2Output(
-		opts, checkpointBlock.Root, timestamp,
+		opts, checkpointBlock.Root, timestamp, [32]byte{}, big.NewInt(0),
```

Comment:
It would be good to do what I've said in the `TODO`, but it's non-essential and I want to focus on withdrawals rather than figure this out.